app/containers/MessageBox/EmojiKeyboard.tsx

```diff
@@ -6,21 +6,23 @@ import { Provider } from 'react-redux';
 import store from '../../lib/store';
 import EmojiPicker from '../EmojiPicker';
 import styles from './styles';
-import { themes } from '../../lib/constants';
-import { TSupportedThemes } from '../../theme';
+import {useTheme} from '../../theme'
+import { EventTypes } from '../EmojiPicker/interfaces';
 
-const EmojiKeyboard = ({ theme }: { theme: TSupportedThemes }) => {
-	const onEmojiSelected = (emoji: string) => {
-		KeyboardRegistry.onItemSelected('EmojiKeyboard', { emoji });
+const EmojiKeyboard = () => {
+	const onItemClicked = (eventType: EventTypes, emoji: string | undefined) => {
+		KeyboardRegistry.onItemSelected('EmojiKeyboard', { eventType, emoji });
 	};
 
+	const {colors} = useTheme()
```

Comment:
Hooks always on top https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Also, you're not using Prettier yet.
Please do it.